### PR TITLE
fix bug

### PR DESCRIPTION
**Pull request by Lafinger**
"G04 9b6bc6ce99b44695a731befc9280ac48,93ff7820620a4e2d96899d85d3806809,10*"
When I read this line in the file, I made an error。

### CODE REVIEW

**Comment by hsiang-lee** [comment]
> "G04 9b6bc6ce99b44695a731befc9280ac48,93ff7820620a4e2d96899d85d3806809,10*" When I read this line in the file, I made an error。

What? I don't understand what you mean.

**Comment by hsiang-lee** [comment]
> "G04 9b6bc6ce99b44695a731befc9280ac48,93ff7820620a4e2d96899d85d3806809,10*" When I read this line in the file, I made an error。

Could you please send you gerber file to me to debug? You can send it by email and my email is leehsiang@sina.com. Thanks!

**Comment by Lafinger** [comment]
I sent it to your email

**Comment by hsiang-lee** [comment]
OK.Thanks!
----- 原始邮件 -----
发件人：Jason Chen ***@***.***>
收件人：hsiang-lee/gerber-parser ***@***.***>
抄送人：Lee Hsiang ***@***.***>,  Comment ***@***.***>
主题：Re: [hsiang-lee/gerber-parser] fix bug (PR #7)
日期：2022年08月31日 14点11分


I sent it to your email

&mdash;

**Review by hsiang-lee — APPROVED** [review]

File: src/gerber/gerber_file.cpp [hunk 1]
```diff
@@ -82,7 +82,7 @@ bool GerberFile::GetInteger(int& integer) {
 
 	bool negative = GetSign();
 	integer = 0;
-	while (!EndOfFile()) {
+	while (pointer_ < buffer_.size()) {
```
[the remark this answers is not in the document]
Comment (Lafinger) [reply src/gerber/gerber_file.cpp:85]: The program prematurely returns when reading "G04 9b6bc6ce99b44695a731befc9280ac48,93ff7820620a4e2d96899d85d3806809,10*" and gerber file parsing failure.
Finally, the program displays blank content .

**Comment by hsiang-lee** [comment]
Give a star if this project helped you!
----- 原始邮件 -----
发件人：Jason Chen ***@***.***>
收件人：hsiang-lee/gerber-parser ***@***.***>
抄送人：Lee Hsiang ***@***.***>,  Comment ***@***.***>
主题：Re: [hsiang-lee/gerber-parser] fix bug (PR #7)
日期：2022年08月31日 14点11分


I sent it to your email

&mdash;

**Pull request merged**